lab_3_generate_by_ngrams/main.py

```diff
@@ -5,6 +5,7 @@
 """
 # pylint:disable=too-few-public-methods
 from typing import Optional
+from string import *
 
```

Comment:
Why use a wildcard? Are there that many string fields you use in your code?

Reply:
I just thought that it seems nice to write import in such way

Reply:
Certainly a fancy one, but useless here